spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/AdminCommunicatorConfiguration.java

```diff
@@ -0,0 +1,88 @@
+package eu.coatrack.proxy.security;
+
+/*-
+ * #%L
+ * coatrack-proxy
+ * %%
+ * Copyright (C) 2013 - 2021 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+
+/**
+ * Builds and provides the required configuration details for communication with the CoatRack admin server.
+ *
+ * @author Christoph Baier
+ */
+
+@Configuration("securityConfiguration")
```

Comment:
I originally wanted to create a central configuration file for the whole security folder but for the moment this configuration bean is limited to the AdminCommunicator class. So the name "securityConfiguration" is currently not fitting. This will be changed.

Reply:
Done.